src/components/IGVFileSelector.js

```diff
@@ -22,10 +22,12 @@ const getStrings = v => {
   )
 }
 
+const MAX_CONCURRENT_IGV_FILES = 10
```

Comment:
@wnojopra do you recall why you limited the number of files that could be opened at a time (the limit was just removed per https://github.com/DataBiosphere/terra-ui/pull/2943)?

Reply:
The details are unfortunately lost in auto-deleted emails, but I recall it was a product decision. I believe there was feedback where if you had a lot of files, you would be easily tempted to open them all up at once. I don't remember the exact behavior of what happens if you do this, but it was fairly unusable, and you ended up downloading a lot of data for nothing.

I don't use IGV at all these days, but there might be some reasonable number for a limit. Perhaps 10 is too low, but I wouldn't want to enable a state where we download a lot of data but end up in an unusable state. 

Or perhaps IGV behaves differently today than it did three years ago, and we don't need a limit anymore!